counter/src/pages/TablePage.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import  './TablePage.css'
+import dayjs from 'dayjs'
+function TablePage(props) {
+  
+   console.log(props.data);
+  return (
+    <div>
+      <table id="student">
+    <tbody>
+      <tr>
+      <th><label for="first">Name</label></th>
+      {/* <th ><label for="last">Last</label></th> */}
+      <th><label for="dob">DOB</label></th>
+      <th><label for="about">About</label></th>
+      </tr>
+    
+      {props.data.map((e, index) => (
+        <tr key={index}>
+          <td>{e.fname}{' '}{e.lname}</td>
+          {/* <td> </td> */}
+          <td>{dayjs(e.date).format('MM/DD/YYYY')}</td>
+          <td>{e.aboutText}</td>
+        </tr>
+      ))}
+    
```

Comment:
format code before to push remove unnecessary space.

Reply:
Removed sir